src/views/Project/index.jsx

```diff
@@ -22,15 +23,58 @@ const tagReposMapping = repositories =>
   }, {});
 
 @hot(module)
-@graphql(issuesQuery, {
-  skip: props => !projects[props.match.params.project].repositories,
-  options: () => ({
-    fetchPolicy: 'network-only',
-    variables: {
-      searchQuery: '',
-    },
+@compose(
+  graphql(issuesQuery, {
+    skip: props => !projects[props.match.params.project].repositories,
+    name: 'github',
+    options: () => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        searchQuery: '',
+      },
+      context: {
+        client: 'github',
+      },
+    }),
   }),
-})
+  graphql(bugsQuery, {
+    skip: props => !projects[props.match.params.project].products,
+    name: 'bugzilla',
+    options: props => ({
+      variables: {
+        search: {
+          tags: ['good-first-bug'],
+          products:
+            projects[props.match.params.project].products.filter(
+              product => typeof product === 'string'
+            ).length > 0
```

Comment:
We can create a function `componentsFromProducts` which takes in a product object and returns an array of components. The latter can return an empty array if the product has no components. To check for emptiness, we can do `!componentsFromProducts(products).length`

As a result logic duplication is removed and the code becomes more readable.